build/apiserver/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian
+FROM centos:7
 
 ADD tmp/apiserver /opt/services/
```

Comment:
Will you carry patches in this repo to change these to be `yum install` stanzas or will you have two Dockerfiles, one for "dev" that does this and one with `yum`?